drf_braces/tests/fields/test_custom.py

```diff
@@ -55,3 +57,16 @@ def test_to_internal_value(self):
 
         self.assertEqual(field.to_internal_value('bar'), 'bar')
         self.assertEqual(field.to_internal_value('haha'), 'haha')
+
+
+class TestCurrencyField(unittest.TestCase):
+    def test_init(self):
+        field = RoundedDecimalField()
+        self.assertIsNone(field.max_digits)
+        self.assertEqual(field.decimal_places, 2)
+
+    def test_to_internal_value(self):
+        field = RoundedDecimalField()
+        self.assertEqual(field.quantize(Decimal('5.2345')), Decimal('5.23'))
```

Comment:
probably better to test the whole validation cycle. you can call `to_internal_value` here. it will then internally call `self.quantize`